catalog/dags/data_refresh/data_refresh_task_factory.py

```diff
@@ -203,8 +210,14 @@ def create_data_refresh_task_group(
         )
         tasks.append(delete_old_index)
 
+        enable_alarms = PythonOperator(
+            task_id="enable_sensitive_cloudwatch_alarms",
+            python_callable=cloudwatch.enable_or_disable_alarms,
+            op_args=[True],
+            trigger_rule=TriggerRule.ALL_DONE,
+        )
         # Finally, promote the filtered index.
-        tasks.append(promote_filtered_index)
+        tasks.append([enable_alarms, promote_filtered_index])
```

Comment:
This also works, but still puts `enable_alarms` in between the promotion/deletion steps for the media index, and the promotion/deletion for the filtered index. I don't think that will cause any problems in terms of task execution, but it makes the flow of the DAG more confusing, and ideally we'd re-enable the alarm ASAP. I'd prefer to define this task earlier and then append it along with the first set of promote tasks on L199.

Reply:
I see I chose the wrong promote task to pair it with. Gotcha, fixed.